src/map/trait.h

```diff
@@ -139,7 +139,11 @@ enum TRAITTYPE
     TRAIT_PRIMEVAL_ZEAL       = 120,
     TRAIT_INSPIRATION         = 121,
     TRAIT_SLEIGHT_OF_SWORD    = 122,
-    TRAIT_DAKEN               = 123
+    TRAIT_DAKEN               = 123,
+    TRAIT_SUPERIOR            = 124, 

```

Comment:
Every job has superior. I'm guessing that the magic accuracy traits are
also from gifts